stdlib/public/core/Int128.swift

```diff
@@ -250,21 +288,23 @@ extension Int128 {
       return (partialValue, overflow || partialValue < 0)
     }
   }
-  
-  @inline(__always)
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
```

Comment:
I'm not certain that we want these to be `@_transparent`. These are potentially pretty heavyweight operations to drop into the caller (though we might reap benefits in the case of constant divisors).